### PR TITLE
Update DBC file and docs

### PR DESCRIPTION
**Pull request by karniv00l**
Used Kvaser V3.21.472

### CODE REVIEW

File: firmware/controllers/can/FOME_CAN_verbose.dbc [hunk 1]
```diff
@@ -1,7 +1,7 @@
-VERSION ""
+VERSION "HIPBNYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYY/4/%%%/4/'%**4YYY///"
```
Comment (mck1117) [inline firmware/controllers/can/FOME_CAN_verbose.dbc:1]: what happened here...?

what version of kvaser database did you use?

Comment (karniv00l) [reply firmware/controllers/can/FOME_CAN_verbose.dbc:1]: 2.5.475, no Idea, let me revert that

Comment (mck1117) [reply firmware/controllers/can/FOME_CAN_verbose.dbc]: update!
image

**Comment by karniv00l** [comment]
Opps, those are unsigned, but set to `-40` and `-50` in the file 😕 

Screenshot 2023-11-03 at 11 24 05

**Pull request merged**